### ISSUE TITLE
Update Event Checklist

### ISSUE THREAD

**Issue by harshkhandeparkar** [kossiitkgp/docs#72]
The following points should be added to/changed in the [Event Checklist](https://github.com/kossiitkgp/docs/blob/1108a05c7b139eda7f84b2f4f19811326e234c17/community/event-checklist.md).
- Updating the [workshops](https://github.com/kossiitkgp/workshops) repository.
- The fact that all workshops are consolidated into the single workshops repo.
> ✔️ Make a Github repo for the workshop with name as event name and year which contains - installation instructions, slides, references, recording and others. (For example refer - [1](https://github.com/kossiitkgp/git-and-github-workshop-2020), [2](https://github.com/kossiitkgp/REST-APIs-in-Flask-Workshop-2021), [3](https://github.com/kossiitkgp/Linux-Shell-Vim-Workshop-2021))

- Anonymous feedback forms.
- Link to the poster design, printing, and pasting guidelines. Also, add a timeline for how early posters should be printed and stuck.
- Link to the room booking guidelines and add a timeline for that too.

**Comment by proffapt** [comment]
It was also decided by Likhith and Rajiv - when I asked - that slides repo should be merged into workshops repo.
- For internal events, there can be a separate repo but separate slides repo makes less sense.
- For things which are not workshops like OSS it was concluded that the name of workshops repo should be changed to "events"

**Comment by harshkhandeparkar** [comment]
Let's do this before any workshops happen then.

**Comment by proffapt** [comment]
I won't be able to do, it's the first thing I put in my "Later" on slack and not done yet, lol.
Can ask a CTM to do..

**Comment by harshkhandeparkar** [comment]
Actually, let's not rush it. Slides repo is mentioned in a lot of places. We will have to update all of those links too.

**Comment by proffapt** [comment]
This was once suggested by Rajiv
might as well mention [The "Three Things" Exercise for getting things out of talk](https://math.stanford.edu/~vakil/threethings.html)

**Comment by proffapt** [comment]
Share post on instagram as well, it isn't mentioned. And you can mention that instagram post require an image/photo/picture whatever.
